tabbycat/tournaments/templates/set_round_weights.html

```diff
@@ -0,0 +1,48 @@
+{% extends "base.html" %}
+{% load static debate_tags add_field_css i18n %}
+
+{% block head-title %}<span class="emoji">⚖️</span> {% trans 'Set Round Weights' context 'page title' %}{% endblock %}
+{% block page-title %}{% trans 'Set Round Weights' context 'page title' %}{% endblock %}
+
+{% block page-alerts %}
+{% blocktrans trimmed asvar p1 %}
+  Round weights are coefficients for the points received in each round.
+  A team would get triple their points obtained in a round with a weight of 3.
+{% endblocktrans %}
```

Comment:
possibly worth saying "team points" instead of just "points" (to be extra unambiguous)